server/views/partials/header.njk

```diff
@@ -20,7 +20,7 @@
         {% if user %}
           <li class="hmpps-header__navigation__item">
             <a data-qa="manageDetails" class="hmpps-header__link" href="/account-details" data-test="manage-account-link" target="_blank" rel="noopener noreferrer">
-              <span data-qa=header-user-name>{{ user.displayName | initialiseName }}</span>
+              <span data-qa="header-user-name">{{ user.displayName | initialiseName }}</span>
```

Comment:
I'm not sure we need initialiseName as all of our user names will be email addresses.

Reply:
This line is the existing block of code that was already there, which is displayed when `user` is present. As I understand it `user` is populated via `populateCurrentUser`, and is essentially when the user has auth'ed and come in via DPS (I might have got some of the words/tech wrong here, but you get my gist). 
I think this will be used by our app when the Mailroom Staff authenticate

The only change I made to this line was to add the quotes around the attribute value, and I only did that for consistency with everything else. Happy to change it back?

Reply:
Oh hang on that's not true - some will be DPS users!

Reply:
Sorry missed your comment before my 2nd one - yes it's fine, my point is rubbish.